fix bus

dengzhongyuan365-dev opened the pull request:
fix: Fix the problem of deformation of the crumb menu.
    
    The rounded corner set by the system is too large. When it is displayed only one item, it cannot be completely drawn to the corner, causing the lower part of the window to abnormal. Compatible treatment will increase height when there is only one item.
    
    Log: Fix the problem of deformation of the crumb menu.
    
    Bug: https://pms.uniontech.com/bug-view-296751.html

dengzhongyuan365-dev committed:
fix: Repair path drop-down menu display problem.

Change the style of selection and suspended style.

Log: Repair path drop-down menu display problem.

Bug: https://pms.uniontech.com/bug-view-296835.html

dengzhongyuan365-dev committed:
fix: Repairing path drop-down menu does not support letters screening, and it cannot be quickly located.

Refer to the logic of the list of the workspace, use the first letter matching to find it when entering a character. When it is Chinese, convert it to Pinyin.

Log: Repairing path drop-down menu does not support letters screening, and it cannot be quickly located.

Bug: https://pms.uniontech.com/bug-view-296839.html

dengzhongyuan365-dev committed:
fix: Fix the problem of deformation of the crumb menu.

The rounded corner set by the system is too large. When it is displayed only one item, it cannot be completely drawn to the corner, causing the lower part of the window to abnormal. Compatible treatment will increase height when there is only one item.

Log: Fix the problem of deformation of the crumb menu.

Bug: https://pms.uniontech.com/bug-view-296751.html

Lighto-Ku commented on src/plugins/filemanager/dfmplugin-titlebar/views/folderlistwidget.cpp:
```diff
@@ -166,7 +166,13 @@ void FolderListWidget::setFolderList(const QList<CrumbData> &datas, bool stacked
+    } else {
+        d->folderView->setViewportMargins(kItemMargin, kItemMargin * 3 / 2, kItemMargin, kItemMargin * 3 / 2);
+        setFixedHeight(kItemMargin * 3 + kFolderItemHeight * folderCount);
+    }
```
这样修改不符合设计预期，只存在一条数据时，上下空白区域太大。

dengzhongyuan365-dev replied:
这里和周倩讨论过，因为系统的圆角在只有一项的时候会绘制不全，出现残角。就还是加高让他显示正常。

Johnson-zs commented on src/plugins/filemanager/dfmplugin-titlebar/views/folderlistwidget.cpp:
```diff
@@ -12,6 +12,7 @@
 #include <QVBoxLayout>
 #include <QStandardItemModel>
 #include <QKeyEvent>
+#include <dfm-base/utils/chinese2pinyin.h>
```
头文件位置没放对

dengzhongyuan365-dev replied:
已经修改

dengzhongyuan365-dev committed:
fix: Fix the sorting button hover effect and inconsistent rounded corners.

Use the same background to draw style with the view button.

Log: Fix the problem of deformation of the crumb menu.

Bug: https://pms.uniontech.com/bug-view-296869.html

dengzhongyuan365-dev committed:
fix: Fix the problem that the text of the tab page is bolded.

Tag page text is not bold.

Log: Fix the problem that the text of the tab page is bolded.

Bug: https://pms.uniontech.com/bug-view-296907.html

dengzhongyuan365-dev committed:
fix: Fix the activation path text is shown boldly and the path text is pure black.

Modify the style expected by the designer.

Log: Fix the activation path text is shown boldly and the path text is pure black.

Bug: https://pms.uniontech.com/bug-view-296895.html
     https://pms.uniontech.com/bug-view-296883.html

Johnson-zs commented:
/forcemerge
